ecf/scripts/workflow_manager/cycled/next_run.ecf

```diff
@@ -0,0 +1,58 @@
+#PBS -S /bin/bash
+#PBS -N %RUN%_next_run_%CYC%
+#PBS -j oe
+#PBS -q %QUEUE%
+#PBS -A %PROJ%-%PROJENVIR%
+#PBS -l walltime=00:30:00
+#PBS -l place=vscatter,select=1:ncpus=1:mem=1GB
+#PBS -l debug=true
+
+model=gfs
+%include <head.h>
+%include <envir-p1.h>
+
+set -x
+
+export NET=%NET:gfs%
+export RUN=%RUN%
+export CDUMP=%RUN%
+
+############################################################
+# Load modules
+############################################################
+
+module list
+
+export edate=%EDATE%
+export cyc=%CYC%
+export cycle=t${cyc}z
+export EXPDIR=${EXPDIR:-$HOMEgfs/parm/config}
```

Comment:
So if `EXPDIR` is not defined at this point, it will fill logs in the `HOMEgfs/parm/config`?
Also, how is ecflow receiving this value for `EXPDIR`? I don' see it defined in `cycled_gfs.def`.
Or is it being set in the environment prior to running `ecflow_client` loading of the suite?

Reply:
Is it being set in the environment manually prior to running ecflow workflow after ecflow_client has loaded the definition file to the server.

Reply:
Ok.  
It would be good to document this in a future PR on how a developer could setup and execute a ecflow workflow.

Reply:
Like the rocoto workflow, ecflow is a workflow. There will be a bit learning associated when developer try to use.